tools/nnyeah/nnyeah/NStrings.Designer.cs

```diff
@@ -0,0 +1,116 @@
+﻿//------------------------------------------------------------------------------
+// <auto-generated>
+//     This code was generated by a tool.
+//     Runtime Version:4.0.30319.42000
+//
+//     Changes to this file may cause incorrect behavior and will be lost if
+//     the code is regenerated.
+// </auto-generated>
+//------------------------------------------------------------------------------
+
+namespace nnyeah {
```

Comment:
@stephen-hawley I did not check on this before but we should use a namespace that follows the .NET conventions on naming https://docs.microsoft.com/en-us/dotnet/standard/design-guidelines/names-of-namespaces

Reply:
@dalexsoto @stephen-hawley  does `Microsoft.Nnyeah` meet all these criteria? Also could this change be in a separate PR?

Reply:
@tj-devel709 @stephen-hawley yeah, let's make this in a separated PR

Reply:
Okay I applied the namespace fix here!